torchdrivesim/simulator.py

```diff
@@ -442,35 +451,48 @@ def _compute_collision_of_multi_agents(self, mask: Optional[Tensor] = None) -> T
         """
         return
 
-    def compute_collision(self) -> TensorPerAgentType:
+    def compute_collision(self, agent_types=[]) -> TensorPerAgentType:
```

Comment:
Same comments as above regarding the type and the default value.

Reply:
Done.